lib/index.js

```diff
@@ -126,18 +155,20 @@ Ramen.prototype.identify = function(identify){
 
   // For tracking purposes
   ramenSettings._partner = 'segment.com';
+  ramenSettings.disable_location_watch = true;
 
   // Expose ramenSettings so Ramen.go() can see it
   window.ramenSettings = ramenSettings;
 
   // Ramen.go() will figure things out if called multiple times
   window.Ramen.go();
+
+  this.identified = true;
 };
 
 Ramen.prototype.group = function(group) {
-  // If there is not already a ramenSettings object, or if Ramen isn't loaded,
-  // then do nothing here.
-  if (!(window.ramenSettings && window.Ramen)) {
+  // If there has not been an identify call yet, do nothing.
+  if (!this.identified) {
```

Comment:
Ok. Does that mean I should be using `this.analytics.user().email()` [here](https://github.com/segment-integrations/analytics.js-integration-ramen/blob/master/lib/index.js#L60) as well?

I've written in the Ramen docs that `identify` needs to be called before anything else will work, but I didn't know about the localstorage option then. So now I'm trying to wrap my brain around the right way to handle the situation where there has been a `group` call but no `identify` call. The thing is, I still want to effectively run all the code that's up in my own `identify` handler because that's where all the magic lives.

Did I build this thing all wrong by putting all the logic in `identify` and not having it in a private function that can be called from the different public methods?